clients/bazel-module-registry/src/main/kotlin/LocalBazelModuleRegistryService.kt

```diff
@@ -83,9 +85,40 @@ class LocalBazelModuleRegistryService(directory: File) : BazelModuleRegistryServ
     override suspend fun getModuleSourceInfo(name: String, version: String): ModuleSourceInfo {
         val sourceJson = moduleDirectory.resolve(name).resolve(version).resolve(SOURCE_JSON)
         require(sourceJson.isFile)
-        return sourceJson.inputStream().use {
+        val sourceInfo = sourceJson.inputStream().use {
             JSON.decodeFromStream<ModuleSourceInfo>(it)
         }
+
+        if (sourceInfo is LocalRepositorySourceInfo) {
+            val moduleBasePathAsFile = File(bazelRegistry.moduleBasePath)
+            val pathAsFile = File(sourceInfo.path)
+
+            when {
+                moduleBasePathAsFile.isAbsolute -> {
+                    logger.warn {
+                        "ORT does not support a Bazel registry module base path with an absolute path: " +
+                            "$moduleBasePathAsFile."
+                    }
+                }
```

Comment:
You can further reduce nesting by writing

                moduleBasePathAsFile.isAbsolute -> logger.warn {
                    "ORT does not support a Bazel registry module base path with an absolute path: " +
                        "$moduleBasePathAsFile."
                }

Similar below.